apps/smart-contracts/core/test/CollateralDepositRecord.test.ts

```diff
@@ -119,51 +120,47 @@ describe('=> CollateralDepositRecord', () => {
-      expect(await depositRecord.getGlobalNetDepositAmount()).to.be.gt(0)
-      expect(await depositRecord.getUserDepositAmount(user.address)).to.be.gt(0)
+    it("should correctly subtract 'amount' from global total when starting from a non-zero value", async () => {
       const globalDepositsBefore = await depositRecord.getGlobalNetDepositAmount()
-      const accountDepositsBefore = await depositRecord.getUserDepositAmount(user.address)
+      const userDepositsBefore = await depositRecord.getUserDepositAmount(user.address)
+      expect(globalDepositsBefore).to.be.gt(0)
+      expect(userDepositsBefore).to.be.gt(0)
 
       await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
 
       expect(await depositRecord.getGlobalNetDepositAmount()).to.eq(
         globalDepositsBefore.sub(TEST_AMOUNT_TWO)
       )
-      expect(await depositRecord.getUserDepositAmount(user.address)).to.eq(
-        accountDepositsBefore.sub(TEST_AMOUNT_TWO)
-      )
+      expect(await depositRecord.getUserDepositAmount(user.address)).to.eq(userDepositsBefore)
     })
 
-    it("should correctly subtract 'amount' from both deposited totals when called again", async () => {
-      await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
+    it("should correctly subtract 'amount' from global total when called again", async () => {
+      await depositRecord.connect(user).recordWithdrawal(user.address, 1)
       const globalDepositsBeforeSecondWithdrawal = await depositRecord.getGlobalNetDepositAmount()
-      const accountDepositsBeforeSecondWithdrawal = await depositRecord.getUserDepositAmount(
+      const userDepositsBeforeSecondWithdrawal = await depositRecord.getUserDepositAmount(
         user.address
       )
 
-      await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_ONE)
+      await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
 
       expect(await depositRecord.getGlobalNetDepositAmount()).to.eq(
-        globalDepositsBeforeSecondWithdrawal.sub(TEST_AMOUNT_ONE)
+        globalDepositsBeforeSecondWithdrawal.sub(TEST_AMOUNT_TWO)
       )
       expect(await depositRecord.getUserDepositAmount(user.address)).to.eq(
-        accountDepositsBeforeSecondWithdrawal.sub(TEST_AMOUNT_ONE)
+        userDepositsBeforeSecondWithdrawal
       )
     })
 
-    it('should set the deposit total to zero instead of underflowing if withdrawal amount is greater than the existing total', async () => {
-      expect(await depositRecord.getGlobalNetDepositAmount()).to.be.eq(testDepositToWithdrawFrom)
-      expect(await depositRecord.getUserDepositAmount(user.address)).to.be.eq(
-        testDepositToWithdrawFrom
-      )
+    it('should set global total to zero instead of underflowing if withdrawal amount is greater than existing total', async () => {
```

Comment:
nit: start using new naming convention if you're updating any tests
```suggestion
    it('sets global deposits to 0 if withdrawal > global deposits', async () => {
```